apps/quasar/app.json

```
@@ -0,0 +1,11 @@
+{
+  "appCategory": "Asset Management",
+  "appName": "Quasar",
+  "appSummary": "Quasar is an appchain for IBC-enabled vaults that automatically execute customizable financial strategies across independent blockchains.",
+  "appWebsiteUrl": "https://www.quasar.fi/",
```

we recommend adding a link to your website where users can try your app right away.
```suggestion
  "appWebsiteUrl": "https://app.quasar.fi/",
```